.github/workflows/check-dockerfile.yml

```diff
@@ -0,0 +1,16 @@
+name: Test Dockerfile build
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    name: Test Dockerfile build
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build the Docker image
+        run: docker build . --file Dockerfile
```

Comment:
Is it possible to use "matrix" feature here and pass different build args for PHP_VERSION? E.g. `8.1,8-fpm-alpine` or `latest`?
```yaml
example_matrix:
    strategy:
      matrix:
        php_version: [8.1,8-fpm-alpine, latest]
```

```shell
docker build --build-arg PHP_VERSION=${{ matrix.php_version }} . --file Dockerfile
```
https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#example-expanding-configurations

Reply:
I would stay with my solution :)